optout/src/main/java/gov/cms/ab2d/optout/gateway/S3GatewayImpl.java

```diff
@@ -62,10 +93,10 @@ public InputStreamReader getOptOutFile() {
     }
 
 
-    private void validateFileName() {
-        if (!FILENAME_PATTERN.matcher(s3Filename).matches()) {
+    private void validateFileName(String fileName) {
+        if (!FILENAME_PATTERN.matcher(fileName).matches()) {
             final String errMsg = "Filename is invalid ";
-            log.error("{} : {} ", errMsg, s3Filename);
+            log.error("{} : {} ", errMsg, fileName);
             throw new RuntimeException(errMsg);
```

Comment:
Similar to the above, let's remove the `throw`, so that we could move on to the next file.

Reply:
hmm... I think this might still be needed here. 
I could change the other catch to `Exception` from `IOException` to handle both these cases....

Reply:
That works too

Reply:
done